kivy/uix/recycleview/views.py

```diff
@@ -139,10 +139,12 @@ class RecycleDataAdapter(EventDispatcher):
     with this instance.
     '''
 
-    # internals
-    views = {}  # current displayed items
-    # items whose attrs, except for pos/size is still accurate
-    dirty_views = defaultdict(dict)
```

Comment:
Don't delete the class declarations, it's required for documentation purposes. And it's always good to declare class variables in their class definitions.